RoslynPluginGenerator/SonarQube.Plugins.Roslyn.PluginGenerator.csproj

```diff
@@ -144,7 +144,9 @@
     <Compile Include="ZipExtensions.cs" />
   </ItemGroup>
   <ItemGroup>
-    <None Include="App.config" />
+    <None Include="App.config">
+      <SubType>Designer</SubType>
+    </None>
```

Comment:
Automatic change by VS; has no effect on the build so might as well just leave it.